modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -102,6 +131,8 @@ public function blockForm($form, FormStateInterface $form_state) {
    */
   public function blockSubmit($form, FormStateInterface $form_state) {
     $this->configuration['dropdown'] = $form_state->getValue('commerce_cart_dropdown');
+    $this->configuration['use_view'] = $form_state->getValue('use_view');
```

Comment:
We should not be storing use_view in configuration. Follow the OrderSummary checkout pane example in how it deals with this (saving view as NULL when the checkbox is unchecked)